live/internal/phx/reply.go

```diff
@@ -139,6 +144,27 @@ func NewHeartbeat(msgRef string) *Reply {
 	}
 }
 
+func NewRedirect(msg Msg, to string) *Reply {
+	redirect, err := json.Marshal(Redirect{
+		To: to,
+	})
+	if err != nil {
+		panic(err) // should never happen
```

Comment:
this can happen if `to` is not valid UTF-8. could that ever happen?

Reply:
`to` is created by calling `url.String()` inside the `Redirect` func. Does appear `url.String()` can return invalid UTF-8 - https://pkg.go.dev/net/url#URL.String

Reply:
ah, good. drop a comment ot that effect?